larq_compute_engine/tflite/tests/bconv2d_int8_test.cc

```diff
@@ -1,116 +0,0 @@
-#include <gtest/gtest.h>
-
-#include "larq_compute_engine/tflite/tests/bconv2d_op_model.h"
-#include "larq_compute_engine/tflite/tests/utils.h"
-
-namespace compute_engine {
-namespace tflite {
-namespace testing {
-
-TEST(BConv2DTests, Int8ErrorDeathTest) {
-  LceTensor<TBitpacked> input_tensor({1, 16, 16, 2});
-  LceTensor<TBitpacked> packed_filter_tensor({128, 3, 3, 64});
-  LceTensor<std::int8_t> post_tensor({128});
-  LceTensor<std::int32_t> threshold_tensor;
-  LceTensor<std::int8_t> output_tensor;
-
-  output_tensor.scale = 1.0f;
-  output_tensor.zero_point = 0;
-
-  // Required for the macro
-  typedef BConv2DOpModel<std::int8_t, std::int8_t> Int8_BConv2DOpModel;
-
-  EXPECT_DEATH(
-      {
-        Int8_BConv2DOpModel m_lce(compute_engine::tflite::Register_BCONV_2D_OPT,
-                                  input_tensor, packed_filter_tensor,
-                                  output_tensor, post_tensor, post_tensor,
-                                  threshold_tensor, 64, 1, 1, Padding_SAME, 0,
-                                  ActivationFunctionType_NONE, 1, 1, 1);
-      },
-      "8-bit quantization is only supported with valid or one-padding.");
-}
-
-TEST(BConv2DTests, Int8PostTest) {
```

Comment:
Should we keep this test and also move it to `bconv2d_test.cc`?

Reply:
I think this test was only testing correctness of Int8 post multiplier/bias. I asked @Tombana and he agreed we could get rid of it.

Reply:
Sounds good